docs/podman-run.1.md

```diff
@@ -181,12 +181,14 @@ detached container with **podman attach**.
 
 When attached in the tty mode, you can detach from the container (and leave it
 running) using a configurable key sequence. The default sequence is `ctrl-p,ctrl-q`.
-You configure the key sequence using the **--detach-keys** option or a configuration file.
-See **config-json(5)** for documentation on using a configuration file.
+You configure the keys sequence using the **--detach-keys** option, or specifying
```

Comment:
I'm not a fan of the word "You" in a man page.  Can you drop it and just cap "Configure"

Reply:
I can drop this "You", note however that there are other "You" around in the doc.